packages/relay/src/lib/eth.ts

```diff
@@ -757,8 +757,8 @@ export class EthImpl implements Eth {
 
     let blockNumber = null;
     let balanceFound = false;
-    let weibars: BigInt = BigInt(0);
-    const mirrorAccount = await this.mirrorNodeClient.getAccountPageLimit(account, requestId);
```

Comment:
Aren't there any mocking changes that need to be done with this no longer being called?

Reply:
I think is called but only on certain cases. when the block is within the last 15 minutes.